react/Payment.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useRef, useEffect } from 'react'
+
+if (window && window.document) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  var postRobot = require('post-robot')
+}
+
+interface Card {
+  cardNumber: string
+  cardName: string
+  cardDate: string
+  cardCvv: string
+}
+
+const IFRAME_APP_VERSION = '0.1.1'
+const LOCAL_IFRAME_DEVELOPMENT = false
+
+let iframeURL = `https://io.vtexpayments.com.br/card-form-ui/${IFRAME_APP_VERSION}/index.html`
+
+if (LOCAL_IFRAME_DEVELOPMENT) {
+  const PORT = 3000
+  iframeURL = `https://checkoutio.vtexlocal.com.br:${PORT}/`
+}
+
+const Payment: React.FC = () => {
+  const [rendered, setRendered] = useState(false)
+  const [iframeData, setIframeData] = useState<Card | null>(null)
```

Comment:
can you rename this? I found it a bit confusing
```suggestion
  const [cardData, setCardData] = useState<Card | null>(null)
```

Reply:
I'm just using it to test the communication between the IO APP and Iframe.